package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@foxglove/message-definition",
-  "version": "0.3.1",
+  "version": "0.4.0",
```

Comment:
We usually do this in a follow-up PR don't we? In case theres some other things merging?